core/src/test/resources/io/grpc/internal/spiffebundle.json

```diff
@@ -0,0 +1,147 @@
+{
+  "trust_domains": {
+    "google.com": {
+      "sequence_number": 123,
+      "keys": [
+        {
+          "x5c": "-----BEGIN CERTIFICATE-----
+                  CORRUPTED_BYTES==
+                  -----END CERTIFICATE-----"
+        }
+      ]
+    },
+    "test.google.com": {
+      "sequence_number": 123,
+      "keys": [
+        {
+          "some_id": "1",
+          "some_key": "some_value"
+        }
+      ]
+    },
+    "test.google.com.au": {},
+    "example.com": {
+      "sequence_number": 12035488,
+      "keys": [
+        {
+          "kty": "RSA",
+          "use": "x509-svid",
+          "x5c": "-----BEGIN CERTIFICATE-----
```

Comment:
Per offline discussions, we need to remove the block headers. Similarly throughout.